### ISSUE TITLE
Add option to include subfolders

### ISSUE THREAD

**Issue by ForgeSource** [nemethviktor/GeoTagNinja#123]
**Is your feature request related to a problem? Please describe.**
GTN only works in a selected folder. There are times when it would be very useful to be able to also include files in sub-folders of that folder, instead of having to to select them one-at-a-time.

**Describe the solution you'd like**
Add an option to include files in sub-folders of the current folder.

**Describe alternatives you've considered**
Selecting those sub-folders one at a time (which is slow and tedious).

**Additional context**
GeoSetter did this with the _File > Flat Mode_ setting.

**Comment by nemethviktor** [comment]
Ok can you have a look at this please -> https://drive.google.com/open?id=13uYyCQE31JlSUb2iamKdYh55bGNqQFOW&usp=drive_fs 

I've tested relatively thoroughly but in all fairness this required a fair bit of code change so can you make a backup of your files first please just in case something goes sideways. Should be okay though but a heads-up.

**Comment by ForgeSource** [comment]
Thank you SO much!

Unfortunately, while the installation seemed to work OK, I'm getting these errors when I try to run it:

![image](https://github.com/user-attachments/assets/dc0b4f0d-06f6-49bd-b7fc-0b3d8cd1b383)

![image](https://github.com/user-attachments/assets/01da065a-0c01-4643-9c9f-81fb26e7db77)

**Comment by nemethviktor** [comment]
Joy. I'm increasingly getting annoyed with the C# SQLite implementation. Any time there's an update to the package something breaks. Lemme have a look at it. 

ps the reason I didn't get this is because I don't actually install the app the same way as users do but I know where the issue is.

**Comment by nemethviktor** [comment]
Right, can you try downloading and installing from the same link as before please.

**Comment by ForgeSource** [comment]
Again, very many thanks.

Yes, this version installed OK (although it did require that I uninstall the existing version and reboot before doing so). It works exactly as I was looking for.

Really appreciate your time and effort proving this enhancement!

**Comment by nemethviktor** [comment]
Yes, I think that's because the build number was the same for the two installs earlier and the one that worked but I'll double check before making it into a proper release.
Glad it works tho - any issues, shout ;)
